python/ray/train/torch/torch_trainer.py

```diff
@@ -22,13 +22,14 @@ class TorchTrainer(DataParallelTrainer):
     The ``train_loop_per_worker`` function is expected to take in either 0 or 1
```

Comment:
In the paragraph above this, it says "already" twice in the sentence -- it would be great to also fix this :)

Reply:
good catch. Fixed